my-menu-tsps-web/src/main/webapp/css/my-menu-tsps.css

```diff
@@ -0,0 +1,34 @@
+/**
+ * Some custom CSS
+ **/
+
+.dashboard-settings.my-menu-tsps {
+  padding: 0;
+}
+
+.my-menu-tsps.col-sm-12 {
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.my-menu-tsps .device-table {
+  padding: 15px;
+  margin-top: 0.5em;
+}
+
+/* Adjust the size of the columns in the table */
+.my-menu-tsps .column-monitor {
+  width: 30%;
+  max-width: 300px;
+  min-width: 80px;
+  font-family: Verdana, Arial, sans-serif;

```

Comment:
Do you really have to set *font-family*?

Reply:
It is the same :D, but just to show that we can customize things ;)

Reply:
Removed